input/pagecontent/change_log.md

```diff
@@ -123,7 +124,7 @@ HL7 specifications use [RFC 2119](https://tools.ietf.org/html/rfc2119) keywords
 ----
 <br/>
 
-**The following changes occurred between [STU 2 publication](http://hl7.org/fhir/us/mcode/STU2/) (January 2022) and the STU 3 ballot (March 2023). For a history of previous changes, please see the prior change logs in the [appropriate versions](http://hl7.org/fhir/us/mcode/history.html).**
+**The following changes occurred between [STU 2 publication](https://hl7.org/fhir/us/mcode/STU2/) (January 2022) and the STU 3 ballot (March 2023). For a history of previous changes, please see the prior change logs in the [appropriate versions](http://hl7.org/fhir/us/mcode/history.html).**
```

Comment:
Earlier you wrote that to see changes prior to STU3, one should go look at STU3 change log...so everything here should be deleted.